toolkit/offchain/src/csl.rs

```diff
@@ -544,7 +570,7 @@ mod tests {
 		assert_eq!(ex_units.steps(), 2000u64.into());
 	}
 
-	fn test_protocol_parameters() -> ProtocolParametersResponse {
+	pub fn test_protocol_parameters() -> ProtocolParametersResponse {
```

Comment:
We have public `test_protocol_parameters` in `test_values` module. I should have perhaps clean it up here. I'll do it. Please use them outside this module, I'll clean up here.